src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/TransformationClient.java

```diff
@@ -58,19 +58,30 @@ public SuccessfulTransformationResponse transformExceptionRecord(
             );
         } catch (HttpStatusCodeException ex) {
             log.error(
-                "Failed to transform Exception Record to case data for case type {} ",
+                "Failed to transform Exception Record to case data for case type {} and id {}",
                 exceptionRecord.caseTypeId,
+                exceptionRecord.id,
                 ex
             );
 
             if (ex.getStatusCode().equals(UNPROCESSABLE_ENTITY) || ex.getStatusCode().equals(BAD_REQUEST)) {
-                throw new InvalidCaseDataException(
-                    ex,
-                    objectMapper.readValue(ex.getResponseBodyAsByteArray(), TransformationErrorResponse.class)
-                );
+                tryParseResponseBody(ex);
```

Comment:
Should be renamed to tryParseResponseBodyAndThrowException or smth like this, otherwise it is not clear that itsonly outcome is throwing exception